source/chapters/configuration.rst

```diff
@@ -183,3 +207,304 @@ already use JACK, it is easiest to let Mixxx suspend PulseAudio and use ALSA.
 .. warning:: On GNU/Linux do *not* use the ``pulse`` device with the ALSA Audio
```

Comment:
I actually do use pulseaudio when I'm doing very basic playback.  I would soften this language.

"On GNU/Linux it is not recommended to use the ``pulse`` device with the ALSA Audio API. This is an emulation layer for ALSA provided by PulseAudio and results in poor performance, although it may be suitable for basic playback.